doc/classes/TransformContainer.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="TransformContainer" inherits="Container" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		Applies transform (offset, rotation, scale) to child controls.
+	</brief_description>
+	<description>
+		[TransformContainer] will transform all its child [Control] nodes according to the specified transform parameters. The control is first stretched to full rect (like in [MarginContainer]) and then the transform is applied on top. You can freely offset, scale and rotate the node, making it useful for animations.
+	</description>
+	<tutorials>
+	</tutorials>
+	<members>
+		<member name="transform_offset" type="Vector2" setter="set_transform_offset" getter="get_transform_offset" default="Vector2(0, 0)">
+			Position offset applied to child controls. If [member transform_offset_relative_to_size] is [code]true[/code], the offset is relative to the node's [member Control.size].
+		</member>
+		<member name="transform_offset_relative_to_size" type="bool" setter="set_transform_offset_relative_to_size" getter="is_transform_offset_relative_to_size" default="false">
+			If [code]true[/code], [member transform_offset] is relative to the child node's [member Control.size], otherwise it is absolute pixel values.
```

Comment:
I know it sounds weird because it's 3 similar words in sequence, but this _should_ be correct because it's referring to the measurement unit (like for "in degrees", "in decibel", etc.).
```suggestion
			If [code]true[/code], [member transform_offset] is relative to the child node's [member Control.size], otherwise it is in absolute pixel values.
```
In another description you actually said the following, which sounds appropriate enough, too:
```suggestion
			If [code]true[/code], [member transform_offset] is relative to the child node's [member Control.size], otherwise it represents absolute pixel values.
```